src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleLinkVerifierScreen.java

```diff
@@ -214,18 +272,27 @@ protected void done() {
     loadingDialog.setVisible(true);
   }
 
-  private LinkedPatrimoine<NamedID> extractInputIds() {
-    List<NamedID> ids = new ArrayList<>();
+  private GoogleLinkList<NamedID> extractInputIds() {
+    List<NamedID> docsIds = new ArrayList<>();
+    List<NamedID> driveIds = new ArrayList<>();
 
     for (JTextField field : inputFields) {
       var rawText = field.getText();
-      var parsedId = linkIdParser.apply(rawText.trim());
-      String urlName = linksData.patrimoineLinkList().get(inputFields.indexOf(field)).name();
-      NamedID namedURL = new NamedID(urlName, parsedId);
-      ids.add(namedURL);
+
+      if (rawText.contains("drive")) {
```

Comment:
if a file is named drive.java and is a google docs link, it will contain drive.
Ensure Text starts with https://drive instead or something similar